example/package.json

```diff
@@ -15,26 +15,27 @@
     "test": "jest"
   },
   "dependencies": {
-    "@react-native-async-storage/async-storage": "^1.15.9",
-    "react": "^17.0.2",
-    "react-native": "^0.66.0"
+    "@react-native-async-storage/async-storage": "^1.17.10",
+    "react": "^18.2.0",
+    "react-native": "^0.71.0-rc.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.15.5",
-    "@babel/runtime": "^7.15.4",
-    "@react-native-community/eslint-config": "^3.0.1",
-    "@react-native-community/eslint-plugin": "^1.1.0",
-    "@types/jest": "^27.5.0",
-    "@types/react": "^17.0.19",
-    "@types/react-native": "^0.64.13",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@babel/core": "^7.20.2",
+    "@babel/runtime": "^7.20.1",
+    "@react-native-community/eslint-config": "^3.2.0",
+    "@react-native-community/eslint-plugin": "^1.3.0",
+    "@tsconfig/react-native": "^2.0.2",
+    "@types/jest": "^29.2.2",
+    "@types/react": "^18.0.25",
+    "@types/react-native": "^0.70.6",
```

Comment:
What happens if you remove this line?

Reply:
@NickGerleman you mean all 4 packages, or just the @types/react-native package?

Reply:
I think he means just the react-native ones; in 0.71 we bundle them into the node_module

Reply:
Yep. Just @types/react-native which should also now be builtin with newer types.

Reply:
Just removed 'em - I think it works! :)